packages/admin/admin/src/form/fields/CheckboxField.tsx

```diff
@@ -12,11 +12,24 @@ export interface CheckboxFieldProps extends FieldProps<string, HTMLInputElement>
 }
 
 export const CheckboxField = ({ fieldLabel, label, componentsProps = {}, ...restProps }: CheckboxFieldProps) => {
-    const { formControlLabel: formControlLabelProps, finalFormCheckbox: finalFormCheckboxProps } = componentsProps;
+    const { formControlLabel: formControlLabelProps, finalFormCheckbox: formCheckboxProps } = componentsProps;
```

Comment:
IMO we should instead have additional checkboxProps. finalFormCheckbox should be deprecated if there's no FinalFormCheckbox anymore. But all the changes in this file could be made in separate PR.

Reply:
Why remove `FinalFormCheckbox` at all? 🤔

Reply:
I guess the task is to remove usages of FinalFormCheckbox 🤔

Reply:
Sorry, maybe this was badly communicated by me... 
What I meant with the task was to simply use the new components where possible, not entirely remove the existing components. 

Of course we can consider fully removing those components but I wouldn't have seen that as part of this refactoring task.

Reply:
> Of course we can consider fully removing those components but I wouldn't have seen that as part of this refactoring task.

Yes, I'd also just adapt the usages, but leave the FinalForm* and *Field components as is.

Reply:
i have changed the component back to the original version